.vscode/settings.json

```diff
@@ -12,5 +12,6 @@
         "mdx"
     ],
     "editor.rulers": [80],
-    "typescript.preferences.importModuleSpecifier": "relative"
+    "typescript.preferences.importModuleSpecifier": "relative",
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
this ensures the TS server uses the workspace version of TS installed in node modules instead of VSCode's

this avoid scenarios where VSCode and rollup show different errors